data/_clean/offDataClean.R

```diff
@@ -0,0 +1,76 @@
+# script to clean offense data
+#setwd("/Users/patelk26/Desktop/projects/Datathon2021/offenses_disposition")
+
+library(tidyverse)
+library(lubridate)
+
+# reading data -----
+sh <- read.csv('statute_hierarchy.csv')
+df <- read.csv('offenses_dispositions_v3.csv')
+
+# remove duplicate rows -------
+df <- df[!duplicated(df),]
+
+# convert "" to NAs -------
+df <- df %>% 
+  mutate_if(is.character, list(~na_if(.,""))) 
+
+# remove rows with disposition == NA -------
+df <- df[!is.na(df$disposition),]
```

Comment:
I would move this chunk below extract_days() and try to pipe everything including other mutates of disposing_authority_fullname, statute_name2, etc.

```suggestion
offense_disp <- offense_disp_raw %>%
  distinct() %>% # remove duplicate rows
  mutate_if(is.character, list(~ na_if(., ""))) %>% # convert "" to NAs
  drop_na(disposition) %>% # remove rows with disposition == NA
  mutate(
    min_period_clean_days = extract_days(min_period),
    max_period_clean_days = extract_days(max_period),
    period_clean_days = case_when(
      is.na(min_period_clean_days) ~ max_period_clean_days,
      is.na(max_period_clean_days) ~ min_period_clean_days,
      TRUE ~ paste(min_period_clean_days, max_period_clean_days, sep = "-"
  ))
```